### PR TITLE
Improve host daemon testing

### PR DESCRIPTION
**Pull request by bn222**

### CODE REVIEW

File: internal/daemon/hostdaemon.go [hunk 1]
```diff
@@ -175,6 +176,7 @@ func (d *HostDaemon) cniCmdDelHandler(req *cnitypes.PodRequest) (*cni100.Result,
 }
 
 func (d *HostDaemon) Listen() (net.Listener, error) {
+	d.wg.Add(1)
```
Comment (wizhaoredhat) [inline internal/daemon/hostdaemon.go:179]: Shouldn't we call wg.Add right before we create the go routine?

Comment (bn222) [reply internal/daemon/hostdaemon.go:179]: Which go-routine?

**Pull request merged**